src/components/task/breadcrumbs.cjsx

```diff
@@ -75,12 +76,12 @@ module.exports = React.createClass
   crumbMounted: ->
     @calculateCrumbsWidth() if @state.crumbsWidth?
 
-  _resizeListener: (sizes) ->
-    shouldShrink = @shouldShrink(sizes)
-    @setState({shouldShrink})
+  didWidthsChange: (prevState, currentState) ->
+    currentState.crumbsWidth isnt prevState.crumbsWidth or currentState.componentEl.width isnt prevState.componentEl.width
```

Comment:
refers to either the width of of the crumbs having changed or the component width having changed